docs/ops/movement/Unique_10.md

```diff
@@ -74,7 +82,7 @@ The operator can either work in elementwise mode searching for unique values in
             <dim>-1</dim>
         </port>
         <port id="4" precision="I32">
-            <dim>-1</dim>
+            <dim>3</dim>
```

Comment:
I think it's very unlikely that just this output gets a static shape while the others are still dynamic. Even though this is just an example I think it should reflect the behavior of the shape inference of this operation.

Reply:
Such improvement was already introduced with reference implementation for Unique.
Please take a look at your changes in the type_prop tests:
https://github.com/openvinotoolkit/openvino/pull/13945/files#diff-18bd27a012b759129bbab5d44c8aa765cf32f76a8d9430bc38026414d81d2171R39
So this update is just to align the spec with the shape inference and reflect current behavior.